src/client/components/TablePaginated/Filters/MultiSelect/MultiSelect.scss

```diff
@@ -0,0 +1,57 @@
+@import 'src/client/style/partials';
+
+$icon-size: 14px;
+
+.filter-multiselect__tooltip-trigger {
+  height: 100%;
+}
+
+div.filter-multiselect__container {
+  border-radius: 2px;
+  border: 1px solid $ui-border;
+  font-size: $font-xxs;
+  min-width: #{3 * $spacing-xl};
+  padding: 0;
+  transition: all 0.2s ease;
+
+  div.select__control {
+    min-height: $spacing-s;
+    padding: 0 6px;
+
+    div.select__indicatorsContainer {
+      div.select__clearIndicator {
+        height: unset;
+        width: unset;
+
+        svg.icon {
+          color: rgba(0, 0, 0, 0.5);
+          height: $icon-size;
+          width: $icon-size;
+        }
+
+        &:hover {
+          background-color: unset;
+
+          svg.icon {
+            color: $ui-destructive;
+          }
+        }
+      }
```

Comment:
It would look like this:

https://github.com/user-attachments/assets/ecd33085-f324-4c6a-bc31-d79e1b5dd23b

If we add it then we should probably change the red color of x button on the selected options, for consistency. 
<img width="842" alt="image" src="https://github.com/user-attachments/assets/e6109ac4-a208-4d33-927c-fe7fec45d4ca">



Current for reference:

https://github.com/user-attachments/assets/03e6b50d-38db-4760-8a3a-a690d9f2f489

Personally, in the selects that are in tables, I like the extra red squared background color.